ci/infra/testrunner/tests/test_masters.py

```diff
@@ -0,0 +1,22 @@
+
+def test_remove_master(deployment, conf, platform, skuba, kubectl):
+    initial_masters = skuba.num_of_nodes("master")
+    new_node_number = initial_masters
+
+    # Make sure we have the expected amount of masters
+    assert skuba.num_of_nodes("master") == conf.skuba.masters
```

Comment:
I think we are abusing of the `skuba` configuration here. We should keel separation of concerns between the testrunner libraries and the tests.